Source/WebGPU/Configurations/WebGPU.xcconfig

```diff
@@ -66,7 +66,10 @@ GCC_PREPROCESSOR_DEFINITIONS_OS_UNFAIR_LOCK_INLINE_ENABLE_WEBGPU_SWIFT = OS_UNFA
 DEFINES_MODULE = YES;
 MODULEMAP_FILE = WebGPU/WebGPU.modulemap;
 
-SUPPORTS_TEXT_BASED_API = YES;
+SUPPORTS_TEXT_BASED_API = $(SUPPORTS_TEXT_BASED_API_($OVERRIDE_SUPPORTS_TEXT_BASED_API));
```

Comment:
Was this intended to be `$(OVERRIDE_SUPPORTS_TEXT_BASED_API)`?

```suggestion
SUPPORTS_TEXT_BASED_API = $(SUPPORTS_TEXT_BASED_API_$(OVERRIDE_SUPPORTS_TEXT_BASED_API));
```

Reply:
Whoops. Yes, will fix.